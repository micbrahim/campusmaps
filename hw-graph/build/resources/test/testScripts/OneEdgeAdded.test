# tests adding an edge, connecting two nodes of a graph, also testing out valid
# parent-child relationship

# create graph
CreateGraph g1

# add two nodes
AddNode g1 n1
AddNode g1 n2

# create and edge between the two nodes
AddEdge g1 n1 n2 e1

# list child of parent node
ListChildren g1 n1