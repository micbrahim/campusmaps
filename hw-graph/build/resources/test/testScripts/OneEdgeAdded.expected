# tests adding an edge, connecting two nodes of a graph, also testing out valid
# parent-child relationship

# create graph
created graph g1

# add two nodes
added node n1 to g1
added node n2 to g1

# create and edge between the two nodes
added edge e1 from n1 to n2 in g1

# list child of parent node
the children of n1 in g1 are: n2(e1)